dev/dll/XamlControlsResources.cpp

```diff
@@ -48,6 +49,9 @@ void XamlControlsResources::UpdateSource()
             if (isInFrameworkPackage)
             {
                 packagePrefix = L"ms-appx://" MUXCONTROLS_PACKAGE_NAME "/" MUXCONTROLSROOT_NAMESPACE_STR  "/Themes/";
+            } else if (isInCBSPackage)
```

Comment:
Needs to be on separate line.
```suggestion
            }
            else if (isInCBSPackage)
```